polkadot/xcm/xcm-executor/src/lib.rs

```diff
@@ -805,7 +813,10 @@ impl<Config: config::Config> XcmExecutor<Config> {
 				.as_mut()
 				.ok_or(XcmError::BadOrigin)?
 				.append_with(who)
-				.map_err(|_| XcmError::LocationFull),
+				.map_err(|e| {
+					log::error!(target: "xcm::process_instruction::descend_origin", "Failed to append Location: {:?}", e);
```

Comment:
```suggestion
					log::error!(target: "xcm::process_instruction::descend_origin", "Failed to append to Location: {:?}", e);
```

Would also be nice to include the location here